x-pack/test/idp-fixture/oidc/override.properties

```diff
@@ -1,4 +1,4 @@
-op.issuer=http://localhost:8080
-op.authz.endpoint=http://localhost:8080/c2id-login/
+op.issuer=http://oidc-provider:8080/c2id
+op.authz.endpoint=http://oidc-provider:8080/c2id-login/
```

Comment:
See https://connect2id.com/products/server/docs/guides/docker for some details. Issuer is the OPs issuer string that it will use to populate the iss claim in the ID Token. The OP also uses this to determine the token  endpoint and userinfo endpoint it expects to be listening on by adding the proper suffix. 

The authz.endpoint is an API endpoint that we use to programmatically authenticate users with the OP.

We use the docker compose service name `oidc-provider` so that es which also runs in docker can access it via it (docker networking magic :) )

Reply:
Sorry for not being clear here. I wasn't asking about `oidc-provider`, which I understand it's docker's doing. Was more about property names, e.g. `op.authz.endpoint` and what do they mean to the OP. The guide you link should answer all my questions. Thanks!